### PR TITLE
Add uploads to packagecloud

### PR DESCRIPTION
**Pull request by Hobbsee**

### CODE REVIEW

File: lib/omnibus-flapjack/publish.rb [hunk 1]
```diff
@@ -198,6 +199,28 @@ def add_to_rpm_repo(pkg, component = 'experimental')
           Mixlib::ShellOut.new('createrepo . 2>&1', :live_stream => $stdout).run_command.error!
         end
       end
+
+      def add_to_packagecloud(pkg, component='experimental')
+        begin
+          credentials = Packagecloud::Credentials.new(ENV['packagecloud_user'], ENV['packagecloud_token'])
```
Comment (jessereynolds) [inline lib/omnibus-flapjack/publish.rb]: It'd be neater to have the credentials picked up from env vars (or wherever) in the Rakefile and then passed through to OmnibusFlapjack::Publish#add_to_packagecloud using an options hash. One reason is to make it easier to test the code.

**Comment by jessereynolds** [comment]
Looks good, apart from that one comment. (Can it really be this simple?)

File: lib/omnibus-flapjack/publish.rb (hunk 1 above)
Comment (jessereynolds) [inline lib/omnibus-flapjack/publish.rb]: Probably also skip the call to add_to_packagecloud if the credentials aren't found in the environment, up in the Rakefile level

Comment (Hobbsee) [inline lib/omnibus-flapjack/publish.rb]: Yep, good thought.  I'll merge this now, and as discussed, move all the env variables to one place in the Rakefile.

**Comment by Hobbsee** [comment]
Yep, apparently!  My upload timed out (from my very slow connection), so we'll see if it actually works, but it looks like it should.

File: lib/omnibus-flapjack/publish.rb (hunk 1 above)
Comment (Hobbsee) [inline lib/omnibus-flapjack/publish.rb]: This is now done.

**Pull request merged**